config/data_centers/data_centers.json

```diff
@@ -139,6 +139,76 @@
     "source": "https://cloud.google.com/compute/docs/regions-zones",
     "operationalSince": null
   },
+  "google-us-central": {
+    "provider": "google",
+    "lonlat": [-94.5786, 39.1],
+    "displayName": "Kansas City, Missouri",
+    "region": "us-central",
+    "zoneKey": "US-MO",
+    "status": "operational",
+    "source": "https://www.google.com/about/datacenters/locations/",
+    "operationalSince": "2019"
+  },
+  "google-asia-southeast": {
+    "provider": "google",
+    "lonlat": [100.4978, 13.7563],
+    "displayName": "Thailand",
+    "region": "asia-southeast",
+    "zoneKey": "TH",
+    "status": "operational",
+    "source": "https://www.google.com/about/datacenters/locations/",
+    "operationalSince": "2022"
+  },
+  "google-australia-southeast": {
+    "provider": "google",
+    "lonlat": [174.7633, -36.8523],
+    "displayName": "Auckland, New Zealand",
+    "region": "australia-southeast",
+    "zoneKey": "NZ",
+    "status": "operational",
+    "source": "https://www.google.com/about/datacenters/locations/",
+    "operationalSince": "2022"
+  },
+  "google-europe-north": {
```

Comment:
There is no region called europe north, there is only europe north1 and europe north2 which are in Finland and Sweden respectively.

Reply:
Thanks I changed it to europe-north2